### PR TITLE
Adding Discussions and new version of Lexi

### PR DESCRIPTION
**Pull request by springcoil**
Here in this pull request we add the new version of Lexi and discussions.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,48 @@
 # Changelog
+## Friday 12th August 2022
+## Docs updates 
+We launched a [Discussion](https://docs.api.audio/discuss) page so you can ask us your questions. And a [FAQs](https://docs.api.audio/discuss?isFAQ=true) page. Check this out!
+
+## Dictionary (Lexi)
```
Comment (LindaKivine) [inline CHANGELOG.md:6]:
```suggestion
## Pronunciation Dictionary (Lexi)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,48 @@
...
+We updated our Dictionary feature - to make it even more user friendly. 
+We call our Dictionary feature - Lexi.
```
Comment (LindaKivine) [inline CHANGELOG.md:8]: Let's remove this part and keep the feature name consistently Pronunciation Dictionary

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,48 @@
 # Changelog
+## Friday 12th August 2022
+## Docs updates 
+We launched a [Discussion](https://docs.api.audio/discuss) page so you can ask us your questions. And a [FAQs](https://docs.api.audio/discuss?isFAQ=true) page. Check this out!
```
Comment (MariaMantalena4) [inline CHANGELOG.md:4]: ''so as you can ask us your questions directly. In addition to that, we have also launched an updated FAQ Page.

File: CHANGELOG.md (hunk 2 above)
Comment (MariaMantalena4) [reply CHANGELOG.md:8]: do you mean to deprecate the wording completely? They will all be called Text Processors in the future

Comment (LindaKivine) [reply CHANGELOG.md:8]: Yeah we shouldnt use LEXI at all. So until we change the name it should be Pronunciation Dictionary

Comment (Sjhunt93) [reply CHANGELOG.md:8]: I could have changed this?. I guess you mean

apiaudio.lexi -> apiaudio.pronuncation?

Comment (LindaKivine) [reply CHANGELOG.md:8]: Yes that would be good if we can also change the routes to that.

**Pull request merged**